frontend/src/components/project_page/pages/ProjectLandingPage/index.tsx

```diff
@@ -7,7 +7,7 @@ const ProjectLandingPage = () => {
   const { project, setProject } = useOutletContext<ProjectPageContext>();
 
   return (
-    <div className="flex flex-col-reverse md:flex-row">
+    <div className="flex flex-col md:flex-row">
```

Comment:
Are we sure we want to remove the reverse layout here? This would put the action buttons for the project page at the bottom of the view for mobile view, rather than at the top when a user first loads the page.

Reply:
added the `-reverse` back